biofefi/pages/4_Train_Models.py

```diff
@@ -99,7 +99,7 @@ def pipeline(
     data = DataBuilder(
         data_path=exec_opts.data_path,
         random_state=exec_opts.random_state,
-        normalization=exec_opts.normalization,
+        normalisation=exec_opts.normalisation,
```

Comment:
```suggestion
        normalisation=exec_opts.normalization,
```
backwards compatibility